index.html

```diff
@@ -279,71 +282,64 @@
-          <a>input media stream track</a>. <a data-lt=
+          track</a>. A <a>video monitor</a> is said to be
+          <dfn>monitoring</dfn>, if the <a>video monitor</a> is associated with
+          an <a>input media stream track</a>. <a data-lt=
           "video monitor event">Video monitor events</a> are dispatched at a
-          <a>monitoring video worker</a>.
+          <a>video monitor</a>.
         </dd>
         <dt>
-          <a>Processing video worker</a>
+          <a><dfn>Video processor</dfn></a>
         </dt>
         <dd>
           Similarly captures an <a>input frame</a> from an <a>input media
           stream track</a>, and in addition, processes the provided <a>output
           frame</a> for use as the <a>source</a> for the <a>output media stream
-          track</a>. A <a>video worker</a> is said to be a <dfn>processing
-          video worker</dfn>, if the <a>video worker</a> is associated with
-          both an <a>input media stream track</a> and an <a>output media stream
-          track</a>. <a data-lt="video processor event">Video processor
-          events</a> are dispatched at a <a>processing video worker</a>.
+          track</a>. A <a>video processor</a> is said to be
+          <dfn>processing</dfn>, if the <a>video processor</a> is associated
+          with both an <a>input media stream track</a> and an <a>output media
+          stream track</a>. <a data-lt="video processor event">Video processor
+          events</a> are dispatched at a <a>video processor</a>.
         </dd>
       </dl>
       <p>
-        An <dfn>input frame</dfn> for a <a>video worker</a> <var>w</var> is an
-        <a>ImageBitmap</a> object representing a frame sourced from the
-        <a>input media stream track</a> associated with <var>w</var>.
+        An <dfn>input frame</dfn> is an <a>ImageBitmap</a> object representing
+        a frame sourced from the <a>input media stream track</a> associated
+        with <a>video monitor</a> or <a>video processor</a>.
       </p>
       <p>
-        An <dfn>output frame</dfn> for a <a>video worker</a> <var>w</var> is an
-        <a>ImageBitmap</a> object assigned to the <a>outputImageBitmap</a>
-        attribute of the <a>VideoProcessorEvent</a> dispatched at <var>w</var>.
+        An <dfn>output frame</dfn> is an <a>ImageBitmap</a> object assigned to
+        the <a>outputImageBitmap</a> attribute of the
+        <a>VideoProcessorEvent</a> dispatched at <a>video processor</a>.
       </p>
       <p>
-        The <dfn>input media stream track</dfn> for a <a>video worker</a>
-        <var>w</var> is a <a>MediaStreamTrack</a> object passed as the first
-        argument to the <code><a>addVideoMonitor</a>()</code> or
-        <code><a>addVideoProcessor</a>()</code> method on <var>w</var>.
+        The <dfn>input media stream track</dfn> is a <a>MediaStreamTrack</a>
+        object passed as the first argument to the
```

Comment:
I am confused with this statement, the first argument of both `addVideoMonitor()` and `addVideoProcessor()` is not a `MediaStreamTrack`, so how could the `input media stream track` be defined in this way?

Should it be something likes this?
```
The <dfn>input media stream track</dfn> is the <a>MediaStreamTrack</a> object on which the <code><a>addVideoMonitor</a>()</code> or <code><a>addVideoProcessor</a>()</code> method is invoked.
```